Weather App/Common/Classes/TextFieldPadding.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  TextFieldPadding.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 02.09.2021..
+//
+
+import Foundation
+import UIKit
+
+class TextFieldPadding: UITextField {
```

Comment:
This class is really not needed. Check the documentation for the `UITextField`.

Reply:
You are right, I will change it.